application/assets/tilemaps/maps/desert/rules.tmx

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's for Tiled's automapping feature. I defined a rule saying that anywhere that there's a pit tile on the background layer, there should be a meta-tile in the drops layer.